docs/source/Tutorials/quickstart.rst

```diff
@@ -59,8 +69,8 @@ you must first install it from FireMarshal:
 
 ::
 
-  ./marshal install workloads/br-base.json
+  ./marshal install br-base.json
 
 This command creates a firesim workload file at
 ``firesim/deploy/workloads/br-base.json``. You can now run this workload using
-the standard FireSim commands. 
+the standard FireSim commands.
```

Comment:
Can we add this back in here? (from what used to be in the firesim docs)

Simply change the
``workloadname`` option in ``firesim/deploy/config_runtime.ini`` to
"fedora-uniform.json" and then follow the standard FireSim procedure for
booting a workload (e.g. :ref:`single-node-sim` or :ref:`cluster-sim`).

Reply:
Fixed in the FireSim docs instead. See https://github.com/firesim/firesim/pull/465 (in Booting-Fedora.rst)